common/lib/common-utils/src/test/mocha/rangeTracker.spec.ts

```diff
@@ -91,8 +91,7 @@ describe("Routerlicious", () => {
 				});
 
 				it("Should be able to create a serialized copy of the range tracker", () => {
-					const serialized = rangeTracker.serialize();
-					assert.ok(serialized);
```

Comment:
Note for reviewers: `serialize` unconditionally returns a defined object with properties, and this is correctly captured by the type-system. This assertion didn't do anything. My assumption is that this is basically a smoke test ensuring that calling serialize doesn't fail altogether. So, I've left it here for now, but it could easily just be removed.